content/en/semi-assembly/_index.md

```diff
@@ -0,0 +1,69 @@
+---
+title: "LumenPnP Semi Assembly"
+linkTitle: "LumenPnP Semi Assembly"
+weight: 5
+type: docs
+resources:
+- src: "**.png"
+- src: "**.jpg"
+menu:
+  main:
+    weight: 5
+---
+
+These are assembly instructions for the **Semi-Assembled** LumenPnP. If you're looking for the instructions for the Kit, you can find them by clicking "LumenPnP Kit Assembly" above.
+
+## What is the LumenPnP?
+
+The LumenPnP is an open source Pick and Place project led by [Opulo](https://www.opulo.io/). The LumenPnP is designed to be low cost and suited towards [mid-scale manufacturing](http://stephenhawes.com/level-2-manufacturing/).
+
+{{< container-image path="semi-hero.png" alt="hero image of LumenPnP" >}}
+
+The LumenPnP is capable of placing parts as small as 0402, with support for down to 0603. It is currently set up to use strip feeders, but development around active, powered feeders is ongoing.
+
+It is completely open source and hackable. The main controller has three AUX ports, allowing anyone to add on peripherals or communicate with other machines.
+
+{{% alert color="danger" title="WARNING" %}}
+This documentation corresponds with a specific machine **Version Number**. The Version Number represents what version of hardware your machine has. If you're building a Semi-Assembled LumenPnP, **you have a v3 machine, and these docs are for you!** You should see "v3.X.X" on the pamphlet that was included in your kit. If you see "v2" on your pamphlet, click on "LumenPnP Kit Assembly" in the menu bar above for instructions for your machine.
```

Comment:
Just link directly to the kit assembly page instead of sending the reader on a quest (even if it's a short one).